pkg/kubectl/cmd/templates/command_groups.go

```diff
@@ -59,3 +66,25 @@ func AddAdditionalCommands(g CommandGroups, message string, cmds []*cobra.Comman
 	}
 	return append(g, group)
 }
+
+func CheckRequiredFlags(cmd *cobra.Command, _ []string) error {
+	flags := cmd.Flags()
+	missingFlagNames := []string{}
+
+	flags.VisitAll(func(flag *pflag.Flag) {
+		requiredAnnotation, found := flag.Annotations[cobra.BashCompOneRequiredFlag]
```

Comment:
this also seems like an inversion... these annotations are for bash completion generation, not to govern behavior when running the command